### PR TITLE
Add actual links to contributor licence agreements

### PR DESCRIPTION
**Pull request by AdamHepner**
Now come on guys, am I the only one around here reading the documentation? :)

But seriously, I was just writing a tiny thank you note about the mug I got from you, and wanted to link to your contributor licence agreement, and couldn't find it in the most expected place of all. If you want it signed at all, then you should make it as easy as possible.

I got the actual links from the mail I had received from @kyleaton back in the day.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Once your changes and tests are ready to submit for review:
...
 
-    {Aspirational:} Please make sure you have signed our Contributor License Agreement. We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
+    {Aspirational:} Please make sure you have signed our Contributor License Agreement (either [Individual Contributor License Agreement v1.0](https://docs.google.com/forms/d/e/1FAIpQLSdA-aWKQ15yBzp8wKcFPpuxIyGwohGU1Hx-6Pa4hfaEbbb3fg/viewform?usp=sf_link) or [Software Grant and Corporate Contributor License Agreement ("Agreement") v1.0](https://docs.google.com/forms/d/e/1FAIpQLSf3RZ_ZRWOdymT8OnTxRh5FeIadfANLWUrhaSHadg_E20zBAQ/viewform?usp=sf_link)). We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
```
Comment (jcampbell) [inline CONTRIBUTING.md]:
```suggestion
    Please make sure you have signed our Contributor License Agreement (either [Individual Contributor License Agreement v1.0](https://docs.google.com/forms/d/e/1FAIpQLSdA-aWKQ15yBzp8wKcFPpuxIyGwohGU1Hx-6Pa4hfaEbbb3fg/viewform?usp=sf_link) or [Software Grant and Corporate Contributor License Agreement ("Agreement") v1.0](https://docs.google.com/forms/d/e/1FAIpQLSf3RZ_ZRWOdymT8OnTxRh5FeIadfANLWUrhaSHadg_E20zBAQ/viewform?usp=sf_link)). We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
```

**Review by jcampbell — COMMENTED** [review]
Love it -- since we've moved on a bit, I'm removing the term aspirational here!

File: CONTRIBUTING.md (hunk 1 above)
Comment (jcampbell) [reply CONTRIBUTING.md]: Great catch, and thank you much! Would you mind pulling the word aspirational (should be one click) and I'll merge!

**Review by jcampbell — APPROVED** [review]

**Comment by jcampbell** [comment]
Thank you!

File: CONTRIBUTING.md (hunk 1 above)
Comment (AdamHepner) [reply CONTRIBUTING.md]: Sure thing

**Pull request merged**